src/domain/services/import.service.ts

```diff
@@ -143,9 +143,9 @@ export class ImportService {
           );
           lastModified = null;
         }
-        // Skip product if nothing has changed and not doing a full load
+        // Skip product if nothing has changed and doing an incremental load
         if (
-          source !== ProductSource.FULL_LOAD &&
+          source === ProductSource.INCREMENTAL_LOAD &&
           lastModified?.getTime() === previousLastModified?.getTime()
```

Comment:
Could you use last_updated_t instead of last_modified_t?  last_updated_t is also changed when update_all_products.pl reanalyzes the product

Reply:
I've made the changes. More files affected but mostly renaming columns